src/components/counter.ts

```diff
@@ -0,0 +1,31 @@
+import {useState, createElement} from 'react';
+
+const Counter = ({initialValue}:CounterProps)=>{
```

Comment:
this is still the functional component. Needs to be a Class component (like this):

`class Counter extends React.Component {

  constructor(props) {
    super(props);
    this.state = {
    };
  }

  render() { }
**}`**

Reply:
![Screenshot (24)](https://github.com/weam1995/ReactMentoringProject/assets/79011727/6cecee55-1d30-413e-b72b-3b26896c91fc)